defaults/main.yml

```diff
@@ -11,11 +11,14 @@ ccache_config_options: {}
 # default user, ansible_user.
 ccache_group: "{{ ansible_user }}"
 
-# Where to install ccache when building from sources
+# Where to install ccache when building from sources or installing precompiled binaries
 ccache_install_prefix: "/usr/local"
 
 # Where/how to install ccache. Valid options include:
 # - build: Build from sources
+# - github: Use a precompiled binary from the ccache GitHub releases page (note: not all
+#   ccache versions support all platforms. Linux binaries started appearing at release
+#   4.6.1, and Mac binaries at release 4.8.)
```

Comment:
Same question here, is there a missing patch version?

Reply:
See above.

Reply:
Fixed in 6da0ef9.